saleor/payment/gateways/braintree/__init__.py

```diff
@@ -141,28 +141,75 @@ def authorize(
         raise BraintreeException(DEFAULT_ERROR_MESSAGE)
 
     gateway_response = extract_gateway_response(result)
+
     error = get_error_for_client(gateway_response["errors"])
     kind = TransactionKind.CAPTURE if config.auto_capture else TransactionKind.AUTH
     credit_card = gateway_response.get("credit_card", {})
+
+    if(error):
```

Comment:
The code should be formatted with Black. The recommended way is to use pre-commit which runs all linters automatically.